tests/fff.test.ts

```diff
@@ -0,0 +1,62 @@
+import { assertEquals } from "../deps/assert.ts";
+import { assertSiteSnapshot, build, getSite } from "./utils.ts";
+import fff from "../plugins/fff.ts";
+
+Deno.test("FFF plugin", async (t) => {
+  const site = getSite({
+    src: "fff",
+    location: new URL("https://example.com/"),
+  });
+
+  site.use(fff({
+    date: "published",
+    presets: [{
+      summary: "content",
+    }],
+    strict: {
+      categories: false,
+      media: {
+        array: false,
+        type: "object",
+      },
+    },
+    getGitDate: true,
+    postTypeDiscovery: true,
+  }));
+
+  // build page
+  await build(site);
+
+  // compare to snapshot
+  await assertSiteSnapshot(t, site);
+
+  const { pages } = site;
+
+  // published => date
```

Comment:
Instead of these manual assertions, it's easier to use the build site to compare these values.
For example, you can use `vto` pages, instead of `md` and print the variables in the pages:

```html
---
tags:
  - foo
  - bar
categories:
  - baz
  - qux
---

FFF Tags Test

The `categories` should be merged into `tags`.

{{ tags.join(", ") }}
{{ categories.join(", ") }}
```

Reply:
> Instead of these manual assertions, it's easier to use the build site to compare these values.

I think assertEquals would be more strict.

Reply:
Well, as you want. I say it because it's easier to maintain.